### ISSUE TITLE
Deployment to TRON

### ISSUE THREAD

**Issue by blasrodri** [0xFableOrg/roll-op#57]
Is this possible at all? And if so, does this script/framework support it?

**Comment by norswap** [comment]
Tron seems to be somewhat EVM compatible: https://developers.tron.network/v4.4.0/docs/vm-vs-evm

I personally would expect it not to work for hard-to-understand reasons due to these differences.

I think the first hurdle would surely be whether Foundry (which is used for compiling contracts & deploying them to the L1) is compatible with Tron.

If that were solved, it would remain to see if the EVM differences in Tron don't break some assumptions in the Optimism L1 contracts.

So tl;dr probably no, could be made to work, but would expect that to be multiple months of work by a very capable person or team.
